webapp/src/shared/components/Input/Input.css

```
@@ -0,0 +1,39 @@
+.form-container {
+  width: 312px;
+}
```

```suggestion
.input {
  width: 100%;
  min-width: 200px;
}
```

suggestion: We could rename this class to `input` to match the naming we have currently (CSS classes match the name of the component).

question: What do we think about adding a width of 100% if we want to allow flexible size? Then we could control the width of this component using a wrapper tag.

---

I change the name and take note for future CSS classes.
Regarding the width, I like the idea, it adds more flexibility for future designs or uses